src/cmd-buildextend-vmware

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+# NOTE: PYTHONUNBUFFERED is set in cmdlib.sh for unbuffered output
+#
+# An operation that mutates a build by generating an ova
+
+import os
+import sys
+import json
+import shutil
+import argparse
+
+sys.path.insert(0, '/usr/lib/coreos-assembler')
+from cmdlib import run_verbose, load_json, write_json, sha256sum_file
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--build", help="Build ID",
+                    required=False)
+args = parser.parse_args()
+
+# default to latest build if not specified
+if not args.build:
+    j = load_json('builds/builds.json')
+    args.build = j['builds'][0]
+
+builddir = os.path.join('builds', args.build)
+buildmeta_path = os.path.join(builddir, 'meta.json')
+buildmeta = load_json(buildmeta_path)
+
+base_name = buildmeta['name']
+vmware_prefix = f'{base_name}-{args.build}-vmware'
+ova_name = f'{vmware_prefix}.ova'
+tmpdir = 'tmp/buildpost-vmware'
+
+if os.path.isdir(tmpdir):
+    shutil.rmtree(tmpdir)
+os.mkdir(tmpdir)
+
+
+def generate_vmware():
+    img_qemu = os.path.join(builddir, buildmeta['images']['qemu']['path'])
```

Comment:
nit: The `buildmeta` content _should_ exist, but may not. It might be worth either checking up front for the existence of the keys or wrapping in a `try`/`except KeyError as e`.